dnsmasq.conf.template

```diff
@@ -0,0 +1,27 @@
+# Template file for dnsmasq generation
```

Comment:
I think you missed the comment here, this should be called `.sample` to make it clear it's a sample can can't be used wholesale. So `dns.masq.conf.tmpl.sample`